components/Advertisement.tsx

```diff
@@ -23,13 +24,14 @@ const Advertisement: React.FC<Props> = ({ img, description, url }) => {
         <p className="text-slate-500 dark:text-slate-400 mb-3">{description}</p>
       </div>
       <div className="flex justify-end p-4 bg-slate-50 dark:bg-slate-800 rounded-b-xl">
-        <a
+        <Link
```

Comment:
curious about this change... shouldn't next Link's be only for links within the site?